contrib/win32/win32compat/misc.c

```diff
@@ -1725,3 +1725,177 @@ get_user_sid(char* name)
 
 	return ret;
 }
+
+/* builds session commandline. returns NULL with errno set on failure, caller should free returned string */
+char* build_session_commandline(const char *shell, const char* shell_arg, const char *command, int pty)
```

Comment:
return type (char *) should alone be on a separate line as per BSD standards.

Reply:
fixed